azure-pipelines-ci.yml

```diff
@@ -0,0 +1,63 @@
+pr: none
+
+# There's a separate pipeline for CI which also uses this file, but with a trigger override in the UI
+# https://dev.azure.com/uifabric/fabricpublic/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=164&view=Tab_Triggers
+trigger: none
+
+variables:
+  - ${{ if not(startsWith(variables['Build.SourceBranch'], 'refs/heads/')) }}:
+      - name: sinceArg
+        value: --since $(targetBranch)
+
+  - group: fabric-variables
+
+  - template: .devops/templates/variables.yml
+
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-latest
+      os: windows # We need windows because compliance task only run on windows.
+    customBuildTags: null # send empty tags since it is not supported on GH repo's for PR's
+    customLogIssues: null # send empty tags since it is not supported on GH repo's for PR's
+    stages:
+      - stage: main
+        jobs:
+          - job: BuildTestLint
+            pool:
+              name: '1ES-Host-Ubuntu'
+              image: '1ES-PT-Ubuntu-20.04'
+              os: linux
+            workspace:
+              clean: all
+            steps:
+              - template: .devops/templates/build-test-lint.yml@self
+
+          - job: DeployE2E
+            displayName: Deploy and E2E
+            workspace:
+              clean: all
+            pool:
+              name: '1ES-Host-Ubuntu'
+              image: '1ES-PT-Ubuntu-20.04'
+              os: linux
+            templateContext:
+              outputs:
+                - output: pipelineArtifact
+                  targetPath: $(System.DefaultWorkingDirectory)/apps/pr-deploy-site/dist
+                  artifactName: web
+            steps:
+              - template: .devops/templates/deployE2E.yml@self
+
+              - script: |
+                  rm apps/pr-deploy-site/dist/public-docsite-v9/storybook/407.13419a99614bf685f100.manager.bundle.js
```

Comment:
what is this ? can we add docs why is this needed ? 

like this hash `407.13419a99614bf685f100` can/will change with any change to the source, which will force us to update this manually every time. can we make this better ?

Reply:
Ah looks like in all the merge logic of do undo, do undo I dropped that.